src/apps/basic-transaction/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+// components
+import useBottomMenuModal from '../../hooks/useBottomMenuModal';
+import TextInput from '../../components/Form/TextInput';
+import Button from '../../components/Button';
+import FormGroup from '../../components/Form/FormGroup';
+import Label from '../../components/Form/Label';
+import Select from '../../components/Form/Select';
+
+// utils
+import { visibleChains } from '../../utils/blockchain';
+
+const App = () => {
+  const [t] = useTranslation();
+  const [destinationAddress, setDestinationAddress] = React.useState<string>('');
+  const [value, setValue] = React.useState<string>('');
+  const [data, setData] = React.useState<string>('');
+
+  const chainOptions = visibleChains.map((chain) => ({
+    id: `${chain.id}`,
+    title: chain.name,
+    value: chain.id,
+  }));
+
+  const [chainId, setChainId] = React.useState<number>(+chainOptions[0]?.value);
+  const { show } = useBottomMenuModal();
+
+  const send = () => {
+    if (!destinationAddress) return;
+    show({
+      type: 'send',
+      payload: {
```

Comment:
Should we add icon support here?

Reply:
Per our Myro drafts we had Identicon here, but we can add Icon as well, dyt we add it now or later? Also I think it has to be optional?

Reply:
Definitely optional, too much overhead for a developer. We can add this later for sure - right now MVP